checkers/commentedOutCode_checker.go

```diff
@@ -18,6 +19,12 @@ func init() {
 	info.Name = "commentedOutCode"
 	info.Tags = []string{linter.DiagnosticTag, linter.ExperimentalTag}
 	info.Summary = "Detects commented-out code inside function bodies"
+	info.Params = linter.CheckerParams{
+		"minLength": {
+			Value: 15,
+			Usage: "min length of comment that can makes the warning trigger",
```

Comment:
'min length of the comment that triggers a warning'

Wdyt?

Reply:
+